### PR TITLE
resubmit react week 1

### PR DESCRIPTION
**Pull request by i-max-xi**
Page ready ✔️ 
Resubmission ✔️

### CODE REVIEW

**Review by Gambit142 — APPROVED** [review]
Hi @i-max-xi 👋,

### Highlights
- README.md is professional ✔️
- Used the `create react app` command to set up the project ✔️
- All dependencies and scripts are present in the `package.json` file ✔️ 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

### Optional Suggestions

N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/Components/Calculator.js [hunk 1]
```diff
@@ -1,3 +1,4 @@
+/* eslint-disable */
 import React, { useState } from "react";
 import "../Styles/Calculator.css";
```
Comment (Kingstalux) [inline src/Components/Calculator.js]: - [ ] Good job so far 👍 You are required to use class-based components here. Kindly refactor your `Calculator.js` component into a class-based component and you'll be good to go 💯

**Review by Kingstalux — CHANGES_REQUESTED** [review]
Hi @i-max-xi, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights 👏 
✅ Good job following the required design 👍 
✅ Linters are passing 👌 
✅ Professional README file 💯 



## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/Components/Calculator.js [hunk 2]
```diff
@@ -1,3 +1,4 @@
...
```
Comment (zhadier) [inline src/Components/Calculator.js]: - [ ] Kindly remove this `eslint-disable` in all of your component files to make sure that linters are working and you are following best practices in all of your components. Please try not to disable linter checks under any circumstances unless you get specific go-ahead from Microverse.

File: src/Tests/Components.test.js [hunk 3]
```diff
@@ -1,5 +1,5 @@
-/* eslint-disable testing-library/prefer-screen-queries */
-/* eslint-disable testing-library/no-node-access */
+/* eslint-disable */
```
Comment (zhadier) [inline src/Tests/Components.test.js]: - [ ] Kindly remove this `eslint-disable` in all of your component files to make sure that linters are working and you are following best practices in all of your components. Please try not to disable linter checks under any circumstances unless you get specific go-ahead from Microverse.

File: src/index.js [hunk 4]
```diff
@@ -1,11 +1,13 @@
-import React from 'react';
-import ReactDOM from 'react-dom/client';
-import './index.css';
-import App from './App';
+/* eslint-disable */
```
Comment (zhadier) [inline src/index.js]: - [ ] Kindly remove this `eslint-disable` in all of your component files to make sure that linters are working and you are following best practices in all of your components. Please try not to disable linter checks under any circumstances unless you get specific go-ahead from Microverse.

**Review by zhadier — CHANGES_REQUESTED** [review]
# Hi @i-max-xi  👋 

Good job so far! 🌤️ 
There are some issues that you still need to work on to go to the next project but you are almost there! 🍓 



## Highlights
✔️ No linter errors
✔️ Git flow was set up correctly

## Required Changes ♻️
-[ ] Please do ensure you follow best practices even if you're resubmitting a project. Kindly ensure that the commit messages portray the changes you made and are a description of the feature you added.

_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question @zhadier so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by KDlamini — APPROVED** [review]
### Hi @i-max-xi  🙋‍♀️,  

Excellent work! 💯 
You have done a great job refactoring your components with React hooks. 
Your feature is complete! There is nothing else to say other than... it's time to merge it. :shipit: 
Congratulations!!!🎊 🥳 

## Highlights 🌞 
✔️  Functional components used instead of classes
✔️  React hooks are used to manage state correctly
✔️ Application works as expected
✔️  No linter errors

# Status: Approved ✔️ 

giphyapproved
 
Cheers 🥂  and Happy coding!!! 👯 
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @KDlamini  in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request closed without merging**